packages/webapp/src/containers/Filter/Animals/index.tsx

```diff
@@ -0,0 +1,183 @@
+  onChange: ContainerOnChangeCallback;
+}
+
+const AnimalsFilterContent = ({
+  animalsFilter,
+  filterRef,
+  filterContainerClassName,
+  onChange,
+}: AnimalsFilterContentProps) => {
+  const { t } = useTranslation(['translation', 'filter']);
+
+  const locations: Location[] = useSelector(locationsSelector);
+
+  const { data: defaultTypes = [] } = useGetDefaultAnimalTypesQuery();
+  const { data: customTypes = [] } = useGetCustomAnimalTypesQuery();
+  const { data: defaultBreeds = [] } = useGetDefaultAnimalBreedsQuery();
+  const { data: customBreeds = [] } = useGetCustomAnimalBreedsQuery();
+  const { data: sexes = [] } = useGetAnimalSexesQuery();
+  const { data: groups = [] } = useGetAnimalGroupsQuery();
+
+  const { handleChange, filteredDefaultBreeds, filteredCustomBreeds } = useVisibleBreeds(
+    defaultBreeds,
+    customBreeds,
+    onChange,
+    filterRef,
+  );
+
+  const filters: ComponentFilter[] = [
+    {
+      subject: t('ANIMAL.FILTER.BATCHES_OR_INDIVIDUALS'),
+      type: FilterType.SEARCHABLE_MULTI_SELECT,
+      filterKey: AnimalsFilterKeys.ANIMAL_OR_BATCH,
+      options: [
+        {
+          value: AnimalOrBatchKeys.ANIMAL,
+          default:
+            animalsFilter[AnimalsFilterKeys.ANIMAL_OR_BATCH][AnimalOrBatchKeys.ANIMAL]?.active ??
+            false,
+          label: t('ANIMAL.FILTER.INDIVIDUALS'),
+        },
+        {
+          value: AnimalOrBatchKeys.BATCH,
+          default:
+            animalsFilter[AnimalsFilterKeys.ANIMAL_OR_BATCH][AnimalOrBatchKeys.BATCH]?.active ??
+            false,
+          label: t('ANIMAL.FILTER.BATCHES'),
+        },
+      ],
+    },
+    {
+      subject: t('ANIMAL.ANIMAL_TYPE'),
+      type: FilterType.SEARCHABLE_MULTI_SELECT,
+      filterKey: AnimalsFilterKeys.TYPE,
+      options: [
+        ...customTypes.map((animalType) => ({
+          value: generateUniqueAnimalId(animalType),
+          default:
+            animalsFilter[AnimalsFilterKeys.TYPE][generateUniqueAnimalId(animalType)]?.active ??
+            false,
+          label: animalType.type,
+        })),
+        ...defaultTypes.map((animalType) => ({
+          value: generateUniqueAnimalId(animalType),
+          default:
+            animalsFilter[AnimalsFilterKeys.TYPE][generateUniqueAnimalId(animalType)]?.active ??
+            false,
+          label: t(`animal:TYPE.${animalType.key}`),
+        })),
+      ],
+    },
+    {
+      subject: t('ANIMAL.ANIMAL_BREED'),
+      type: FilterType.SEARCHABLE_MULTI_SELECT,
+      filterKey: AnimalsFilterKeys.BREED,
+      options: [
+        ...filteredCustomBreeds.map((animalBreed) => ({
+          value: generateUniqueAnimalId(animalBreed),
+          default:
+            animalsFilter[AnimalsFilterKeys.BREED][generateUniqueAnimalId(animalBreed)]?.active ??
+            false,
+          label: animalBreed.breed,
+        })),
+        ...filteredDefaultBreeds.map((animalBreed) => ({
+          value: generateUniqueAnimalId(animalBreed),
```

Comment:
I'm not sure where we landed on this one, but would using the `key` for default breeds and the `name` for custom breeds be an option here for value so that we don't have to prefix?

Reply:
I remember discussing, but I'm not sure we decided. I think we also said that to go that route we need a check on the backend to make sure that a user doesn't create a custom that duplicates an existing default (although I guess it would have to be in English and in all caps to actually create a conflict?) and since we didn't get into details of what and when to enforce that, I left it!

I think we will need that check before animals is released anyway though, so we can add the frontend change now. Should it apply to types (and the KPI) as well?

Reply:
I'm good with keeping as is for now!